sdk/cognitiveservices/azure-ai-inkrecognizer/azure/ai/inkrecognizer/_enums.py

```diff
@@ -0,0 +1,194 @@
+    InkStrokeKind.UNKNOWN.
+    """
+
+    DRAWING = "inkDrawing"
+    """ Specify the stroke is a drawing stroke. """
+
+    WRITING = "inkWriting"
+    """ Specify the stroke is a writing stroke. """
+
+    UNKNOWN = "unknown"
+    """ Use default stroke kind in ApplicationKind. If ApplicationKind set to
+    Unknown, server will recognize the stroke kind. """
+
+
+class InkPointUnit(Enum):
+    """ The length unit passes to InkPoint. Default is InkPointUnit.MM. """
+
+    MM = "mm"
+    """"""
+
+    CM = "cm"
+    """"""
+
+    INCH = "in"
+    """"""
+
+
+class InkRecognitionUnitKind(Enum):
+    """
+    The InkRecognitionUnitKind has all the different categories of recognition
+    units available from the service.
+    """
+
+    UNKNOWN = "unknown"
+    """
+    Some stroke can't recognized by Ink Recognizer service.
+    """
+
+    INK_BULLET = "inkBullet"
+    """
+    A bullet on a line of text. The bullet can be associated with more than
+    one line.
+    """
+
+    INK_WORD = "inkWord"
+    """ A word which may have alternates, or an alternate of a word. """
+
+    INK_DRAWING = "inkDrawing"
+    """ A drawing which may have alternates, or an alternate of a drawing. """
+
+    PARAGRAPH = "paragraph"
+    """ A paragraph that contains multiple lines and / or lists. """
+
+    LINE = "line"
+    """
+    A line, that contains a list of words, and may contain a bullet if it
+    inside a list.
+    """
+
+    WRITING_REGION = "writingRegion"
+    """
+    A writing region is a part of the writing surface that contains words.
+    """
+
+    LIST_ITEM = "listItem"
+    """
+    One item of a list. It has only a line as its child, where a bullet should
+    at the begining of that line.
+    """
+
+
+class ShapeKind(Enum):
+    """
+    The Shape enum represents different shapes that can be reported by the ink
+    recognizer service. Any unrecognized shpae will be reported as
+    ShapeKind.DRAWING.
+    """
+
+    BLOCK_ARROW = "blockArrow"
+    """"""
+
+    CIRCLE = "circle"
+    """"""
+
+    CLOUD = "cloud"
+    """"""
+
+    CURVE = "curve"
+    """"""
+
+    DIAMOND = "diamond"
+    """"""
+
+    DRAWING = "drawing"
+    """"""
+
+    ELLIPSE = "ellipse"
+    """"""
+
+    EQUILATERAL_TRIANGLE = "equilateralTriangle"
+    """"""
+
+    HEART = "heart"
+    """"""
+
+    HEXAGON = "hexagon"
+    """"""
+
+    ISOSCELES_TRIANGLE = "isoscelesTriangle"
+    """"""
+
+    LINE = "line"
+    """"""
+
+    PARALLELOGRAM = "parallelogram"
+    """"""
+
+    PENTAGON = "pentagon"
+    """"""
+
+    POLYLINE = "polyline"
+    """"""
+
+    QUADRILATERAL = "quadrilateral"
+    """"""
+
+    RECTANGLE = "rectangle"
+    """"""
+
+    RIGHT_TRIANGLE = "rightTriangle"
+    """"""
+
+    SQUARE = "square"
+    """"""
+
+    STAR_CROSSED = "starCrossed"
+    """"""
+
+    STAR_SIMPLE = "starSimple"
+    """"""
+
+    TRAPEZOID = "trapezoid"
+    """"""
+
+    TRIANGLE = "triangle"
+    """"""
+
+
+class ServiceVersion(Enum):
```

Comment:
What will you do once you have a v1.0 preview version?

Reply:
We are in v1.0 preview now. 
The "preview" string here is to hide the real sub-url from user, only expose a "preview" string as a comment for the version.
Also, as Elijah says, we will update SDK each time our cognitive service has new version.

Reply:
change "preview" into "v1.0-preview"